### PR TITLE
[17.0][FIX] stock_picking_invoice_link: pass parameter to super

### PR DESCRIPTION
**Pull request by ferran-S73**

### CODE REVIEW

File: stock_picking_invoice_link/wizards/account_move_reversal.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def reverse_moves(self, is_modify=False):
         """Link return moves to the lines of refund invoice"""
         action = super(
             AccountMoveReversal, self.with_context(force_copy_stock_moves=True)
-        ).reverse_moves()
+        ).reverse_moves(is_modify)
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/wizards/account_move_reversal.py]: Don't lose keyword arguments

```suggestion
        ).reverse_moves(is_modify=is_modify)
```

File: stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py [hunk 2]
```diff
@@ -272,7 +272,7 @@ def test_invoice_refund(self):
         )
         wiz_invoice_refund.modify_moves()
         new_invoice = self.so.invoice_ids.filtered(
-            lambda i: i.move_type == "out_refund" and i.state == "draft"
+            lambda i: i.move_type == "out_refund"
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py:275]: This change doesn't seem related.

Comment (ferran-S73) [reply stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py:275]: The refund move should be posted, however not passing the is_modify parameter was causing it to stay in draft state,

File: stock_picking_invoice_link/wizards/account_move_reversal.py (hunk 1 above)
Comment (ferran-S73) [reply stock_picking_invoice_link/wizards/account_move_reversal.py]: Done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
OK then

**Comment by pedrobaeza** [comment]
Superseded by #1712, sorry.
